%% ------------------------------------------------------------------------- %%
\chapter{ADAPTAÇÃO LOCAL AOS ANDES}
\label{chap:andes}

\section{INTRODUÇÃO}
\label{sec:andes_intro}

\subsection{Características e Populações Nativas}

A Cordilheira dos Andes, ou simplesmente Andes, trata-se de uma vasta cadeia montanhosa que se estende desde a Venezuela até a Patagônia (aproximadamente 8.000 km), e corresponde a uma das principais ecorregiões da América do Sul. Sua altitude média é de 4.000 m acima do nível do mar. Na região central com elevada altitude, há o planalto Altiplano, abrangendo três países: Peru, Bolívia e Chile. De modo geral, a região ao nordeste dos Andes é mais baixa, úmida e fértil, enquanto existe uma região ao sudeste mais alta e árida. 

As diferenças nas latitudes andinas são importantes para os estudos de estrutura populacional, migração e interações culturais e socioeconômicas entre outras regiões, como a Amazônia \cite{borda_genetic_2020}.

As populações mais estudadas e também as mais abundantes nas terras altas andinas são os povos indígenas Quéchua e Aymara, com aproximadamente 6 milhões de habitantes que residem majoritariamente no Peru e Bolívia \cite{julian_human_2019}.  Há evidências arqueológicas da ocupação humana neste ambiente há cerca de 14.000 AP \cite{rademaker_paleoindian_2014}.

Estudos fisiológicos revelaram as populações das terras altas possuem maior concentração de hemoglobina no sangue, em contraste com populações próximas de terras baixas, ou mesmo em populações de terras altas da Etiópia e Tibete, que só apresentaram tal aumento de concentração quando acima de 4.000 m de altitude \cite{jeong_adaptations_2014}. Outros fenótipos identificados na população nativa dos Andes são aumento da pressão pulmonar arterial, aumento da ventilação em repouso, diminuição da saturação arterial de oxigênio e presença de vasoconstrição devido à hipóxia \cite{jeong_adaptations_2014}.

Ao passo que o aumento da concentração de hemoglobina no sangue fornece uma vantagem para circulação e distribuição de oxigênio aos tecidos do corpo, há como consequência maior viscosidade sanguínea, que por sua vez pode gerar complicações para o sistema circulatório e o período de gestação.

\subsection{Adaptação Local aos Andes}

Considera-se “terras altas” ou “elevadas altitudes” as regiões com mais de 2.500 metros acima do nível do mar (MOORE, 2017). Além das terras altas Andinas, estudos de seleção são frequentemente conduzidos nas terras altas do Himalaia (China/Tibete) e Etiópia (África), uma vez que a pressão seletiva nestas regiões é similar, além de apresentarem populações nativas isoladas durante milhares de anos, tornando-os “laboratórios” excelentes para análise de seleção \cite{fan_going_2016}. 

Assim como os estudos com pigmeus e outras populações nativas de floresta tropical podem ser utilizadas para comparação com os estudos na Amazônia \cite{amorim_detection_2015}, estudos realizados nas terras altas do Tibete e Etiópia costumam ser consideradas em conjunto no estudo de adaptação local aos Andes \cite{bigham_human_2014}, quer seja para verificar convergência evolutiva, ou evolução em diferentes loci que contribuem para um mesmo fenótipo, ou mesmo distintos fenótipos para lidar com uma mesma pressão evolutiva. Embora existam mais estudos nos Andes do que na Amazônia, o mesmo não é verdade para regiões com pressões seletivas similares, como nas terras altas do Tibete \cite{julian_human_2019}.

A maioria das análises de seleção realizadas nas populações nativas dos Andes tem identificado genes pertencentes a via dos fatores indutores de hipóxia (HIF, do inglês, \textit{hypoxia-inducible factors}) ─ uma via essencial na homeostase do oxigênio ─, ainda que estes não sejam os únicos responsáveis pela homeostase do oxigênio,  tampouco os únicos alvos de seleção nestes ambientes (revisado por \citeauthor{julian_human_2019}, \citeyear{julian_human_2019}). De fato, \citeonline{bigham_identifying_2009} verificaram que a via HIF, como um todo, não apresenta sinal de seleção \cite{bigham_identifying_2009}.

Se comparado com outros povos nativos de terras altas, como, por exemplo, as populações do Himalaia, os nativos Andinos apresentam tanto sinais de convergência evolutiva, como, majoritariamente, sinais de seleção particulares dos Andes. Um dos sinais de convergência evolutiva mais fortes foi encontrado no gene \textsl{EGLN1} \cite{bigham_identifying_2010}, que codifica uma molécula sensitiva ao oxigênio responsável pela regulação da transcrição de HIF \cite{to_suppression_2005}. Pelo menos outros 18 genes foram detectados em convergência evolutiva com populações nativas das terras altas do Himalaia, conforme revisto por \citeonline{moore_human_2017}.

Dentre os genes candidatos identificados em populações Andinas, destacam-se genes envolvidos na angiogênese (\textsl{VEGF}, \textsl{ELTD1}) \cite{eichstaedt_andean_2014}, vaso-reatividade (\textsl{EDNRA}, \textsl{BRINP3}, \textsl{NOS2}, \textsl{PRKAA1}, \textsl{TBX5}) \cite{bigham_identifying_2009, bigham_identifying_2010, crawford_natural_2017} e defesa oxidativa (\textsl{FAM213A}) \cite{valverde_novel_2015}. Ademais, foram detectados sinais de seleção positiva no gene \textsl{AS3MT}, responsável pela metabolização de arsênico, uma substância tóxica comum em fontes de água em regiões Andinas (\emph{e.g.} Puna) \cite{eichstaedt_positive_2015}, bem como no gene \textsl{TMEM38B}, que possivelmente atua na redução da policitemia (excesso de eritrócitos, com consequente aumento da viscosidade e prejuízo no fluxo sanguíneo) \cite{crawford_natural_2017}.

A fim de detectar possíveis novos candidatos para adaptação local nas terras elevadas andinas, utilizamos populações nativas publicadas no artigo de \citeonline{reich_reconstructing_2012} e aplicamos estatísticas inferenciais de seleção, como XP-EHH e PBS, simulação de cenários demográficos e análise de expressão gênica in silico. Como resultado, publicamos um artigo na revista \textit{Scientific Reports}, sendo a primeira autoria dividida entre Vanessa Jacovas, Cainã Max Couto-Silva e Kelly Nunes \cite{jacovas_selection_2018}. Atualmente, o artigo conta com sete citações e um dos genes candidatos (\textsl{DUOX2}), identificado pela primeira vez em nosso artigo, também foi detectado sob seleção nos Andes em artigo posterior \cite{borda_genetic_2020}.

\section{ARTIGO}
\label{cap3:andes_artigo}

\selectlanguage{english}

\subsection*{Abstract}

The Andean Altiplano has been occupied continuously since the late Pleistocene, ~12,000 years ago, which places the Andean natives as one of the most ancient populations living at high altitudes. In the present study, we analyzed genomic data from Native Americans living a long-time at Andean high altitude and at Amazonia and Mesoamerica lowland areas. We have identified three new candidate genes - \textsl{SP100}, \textsl{DUOX2} and \textsl{CLC} - with evidence of positive selection for altitude adaptation in Andeans. These genes are involved in the \textsl{TP53} pathway and are related to physiological routes important for high-altitude hypoxia response, such as those linked to increased angiogenesis, skeletal muscle adaptations, and immune functions at the fetus-maternal interface. Our results, combined with other studies, showed that Andeans have adapted to the Altiplano in different ways and using distinct molecular strategies as compared to those of other natives living at high altitudes.

\newpage
\subsection{Introduction}

Along their great expansion, humans have inhabited almost all environments in the five continents. Among several harsh environments that were occupied, the highlands are probably the ones that needed more adaptations for survival \cite{espinoza-navarro_effects_2011}. At least in three geographically distinct locations have this evolutionary adaptation been studied: Andean Altiplano (South America), Himalaya (China/Tibet, Asia) and Semien Mountain (northern Ethiopia, Africa) Plateaus. Andes have been peopled continuously since the late Pleistocene, ~12,000 yBP2 while the time of settlement and permanent occupation of both Tibet and Ethiopia remain a topic of debate, varying widely \cite{aldenderfer_peopling_2011,lu_ancestral_2016}. Despite some uncertainties in the permanent occupation dating, it is certain that humans have inhabited these regions of hostile climates for thousands of years.

Several physiologic factors are associated with living at high altitude ($\geq$ 2,500 meters where only 75\% of the oxygen available at sea level occurs; (\url{http://www.altitude.org/air\_pressure.php}), including adaptations for high ultraviolet radiation index, thermal amplitude, and changes in the pulmonary capacity due to hypoxia \cite{moore_human_2001,sabeti_genome-wide_2007}. High altitude leads to a rapid physiologic/adaptive response in individuals from lowlands; however, prolonged exposure to environmental-related factors might have harmful outcomes. Remarkable features such as increased pulmonary function, hypoxia tolerance, and increased hemoglobin levels have been observed in Andean populations \cite{bigham_human_2014}. How such adaptations took place is still not clear, and just a few genes have been associated with the high altitude adaptation phenotype in human populations \cite{scheinfeldt_genetic_2012,huerta-sanchez_genetic_2013,simonson_adaptive_2015,valverde_novel_2015,fehren-schmitz_ancient_2016,crawford_natural_2017}.

Interestingly, the set of genes presenting signs of natural selection changes according to high altitude, indicating that under an analogous selective pressure, different genetic solutions have emerged. For instance, genomic scans for selection have revealed at least 40 candidate genes related to the Hypoxia Inducible Factor (HIF), such as \textsl{EPAS1} in populations from Tibet, \textsl{EGLN1} in Andeans and Tibetans and \textsl{THRB} and \textsl{ARNT2} in Ethiopians \cite{beall_natural_2010,bigham_identifying_2010,simonson_genetic_2010,peng_genetic_2011,xu_genome-wide_2011,scheinfeldt_genetic_2012}. The populations from the Andean plateau also presented signs of natural selection in other genes, such as \textsl{BRINP3}, \textsl{NOS2}, and \textsl{TBX5}, involved in the nitric oxide pathway (NOS) and related to cardiovascular health \cite{fehren-schmitz_ancient_2016}. In addition, \citeonline{jacovas_genetic_2015} using the candidate gene approach inferred that a combination of some derived and ancestral alleles of \textsl{USP7}, \textsl{LIF} and \textsl{MDM2} genes, all three in the \textsl{TP53} pathway, could have been essential for the successful establishment of Native American populations in the Andean highlands.

Since different investigations pointed to distinct sets of genes involved in high altitude adaptation, more studies are necessary to fully understand the different genetic landscapes present in highland populations around the world. In the present study, we compared genomic data from Native American populations living for a long-time at high altitude (Andean Altiplano) with those living at lowlands (Amazon and Mesoamerica), with the purpose of expanding our knowledge about the genetic repertoire responsible for the successful human colonization of the Andes.

\subsection{Results}

\subsubsection{Natural selection analysis}

Population Branch Statistic (PBS) values were estimated for each individual SNP. To avoid spurious results due to single SNPs, windows of 20 SNPs were used to estimate the mean PBS values for a given region. Then, we checked the outliers’ peaks, above the 99.5th and 99.9th percentiles, to identify in each outlier window the SNPs with the highest PBS value and assigned the gene to which it belonged (or the nearest gene). Based on this approach, five candidate genes were identified: \textsl{SP100} (SP100 Nuclear Antigen), \textsl{TMEM38B} (Transmembrane Protein 38B), \textsl{AS3MT} (Arsenite 3 Methyltransferase), \textsl{DUOX2} (Dual Oxidase 2) and \textsl{CLC} (Charcot-Leyden Crystal Galectin, also known as Galectin-10) (Table \ref{tab:SciRep_table1} and Figure \ref{fig:SciRep_fig1}). Among these candidate genes, \textsl{AS3MT} and \textsl{TMEM38B} have been identified in previous scans for natural selection in Andeans \cite{eichstaedt_positive_2015,crawford_natural_2017}.

\vspace{\onelineskip}

% Tabela 3.1
\input{tabelas/SciRep_table1}

% Figura 3.1
\begin{figure}[ht]
\centering
\includegraphics{SciRep_fig1}
\caption[Average PBS values in windows of 20 SNPs, using a step size of 5 SNPs]{Average PBS values in windows of 20 SNPs, using a step size of 5 SNPs. The 99.5th and 99.9th percentiles of the empirical distribution are shown as black dashed horizontal lines. Names of genes associated with the highest peaks are shown.}
\label{fig:SciRep_fig1}
\end{figure}

\newpage

Neutral coalescent simulations indicated that these deviations were statistically significant (\emph{p}-values ranging between 0.03 and 0.0001; Figure \ref{fig:SciRep_fig2}, Table \ref{tab:SciRep_tableS1}), consistent with the action of positive selection as opposed to genetic drift in increasing the frequency of the putative selected alleles at all five tested loci. In addition, we applied the Cross-Population Extended Haplotype Homozygosity (XP-EHH) test to the same regions. The XP-EHH results also show significant differences between the Andean and Mesoamerican groups in three SNPs (rs13411586, rs9678342, rs7582700) of \textsl{SP100} and one SNP (rs269866) of \textsl{DUOX2} (Table \ref{tab:SciRep_table1}). These SNPs, which are under putative selection in the PBS analysis with the most extreme values (0.46 to 0.62), also present significant XP-EHH values $\geq$ 2 in both Andean vs Mesoamerican and Andean vs Amazonian groups.

% Figura 3.2
\begin{figure}[p] % !ht % H
\noindent
\centering
\includegraphics[width=0.55\linewidth]{SciRep_fig2}
\caption[Distribution of 10,000 simulated PBS values under three neutral coalescent models]{Distribution of 10,000 simulated PBS values under three neutral coalescent models. \textbf{(a)} Constant population model. \textbf{(b)} Population bottleneck model; and \textbf{(c)} Population bottleneck followed by expansion model. The dashed line represents the top observed PBS SNP values in the empirical datasets.}
\label{fig:SciRep_fig2}
\end{figure}

The observed allele density provided by the iHS test showed a notable Gaussian distribution pattern for all three groups (Supplementary Figure \ref{fig:SciRep_figS1}), with homozygosity decaying according to the distance from the focal markers.

It should be noted that the distribution of alleles C (rs13411586, \textsl{SP100}), G (rs269866, \textsl{DUOX2}) and G (rs440191, \textsl{CLC}), which presented the highest PBS values (Table \ref{tab:SciRep_table1}), showed their highest values in areas of very high Andean altitudes (Table \ref{tab:SciRep_table2} and Figure \ref{fig:SciRep_fig3}).

% Table2
\input{tabelas/SciRep_table2}

% Figure 3
\begin{figure}[p] 
\noindent
\centering
\includegraphics[width=0.69\linewidth]{SciRep_fig3}
\caption[Target allele frequencies in South America]{\textbf{(a)} rs13411586\_C (\textsl{SP100}). \textbf{(b)} rs269866\_G (\textsl{DUOX2}) and \textbf{(c)} rs440191\_A (\textsl{CLC}) allele frequency distributions according to altitude. Populations (n $\geq$ 3): 1. Aymara, 2. Guahibo, 3. Guarani, 4. Kaqchikel, 5. Karitiana, 6. Kogi, 7. Maya, 8. Mixe, 9. Mixtec, 10. Piapoco, 11. Quechua, 12. Surui,13. Tepehuano, 14. Ticuna, 15. Toba, 16. Wayuu, 17. Wichi, 18. Yaghan and 19. Zapotec.}
\label{fig:SciRep_fig3}
\end{figure}

\newpage

Bootstrap simulations indicated that in all instances the 95\% confidence interval of allele frequencies in lowlanders does not include the average values observed for populations living in high altitudes (>4000 m above sea level) (Supplementary Figure \ref{fig:SciRep_figS2}), suggesting that the differences found in allele frequencies between population groups might be caused by a non-random evolutionary process.

\subsubsection{Effects of putatively selected alleles on gene expression}

Homozygotes for the \textsl{DUOX2} putatively selected allele (rs269866 G) presented a slight increase in the expression of the \textsl{DUOX2} protein (Supplementary Figure \ref{fig:SciRep_figS3}). Multiple testing across tissues showed significant expression of this protein in thyroid (m-value = 1.0), lungs (m-value = 0.996) and aorta artery (0.996) (Supplementary Figure \ref{fig:SciRep_figS4}). Homozygotes for the rs13411586 (\textsl{SP100}) putatively selected allele (C) presented an increase in the expression of the SP100 protein in skeletal muscles (Supplementary Figure \ref{fig:SciRep_figS5}). Multiple testing across tissues showed significant expression of this protein in skeletal muscle (m-value = 1.0) and testis (m-value = 0.971) (Supplementary Figure \ref{fig:SciRep_figS6}). There is no information available about the \textsl{CLC} gene expression profile.

\subsection{Discussion}

We identified five loci under positive selection in Andean Native populations. Two of them were previously described: \textsl{AS3MT} was found to be under positive selection in Colla Andeans systematically exposed to arsenic water \cite{eichstaedt_positive_2015} while \textsl{TMEM38B} reduced the negative effects of polycythemia (elevated hematocrit or decreased plasma volume) at high altitudes \cite{crawford_natural_2017}. Three other genes, \textsl{SP100}, \textsl{DUOX2}, and \textsl{CLC} were identified for the first time in a high-altitude context in the present study. These genes are part of the \textsl{TP53} pathway, already indicated as a potential candidate to be under natural selection in high altitude populations \cite{eichstaedt_andean_2014,jacovas_genetic_2015}.

\textsl{SP100} is a single-copy gene in the human genome that produces several alternatively spliced Sp100 protein isoforms known as modulators of the p53 activity \cite{berscheminski_sp100a_2016}. We found three SNPs in the \textsl{SP100} gene with high and significant PBS values, as well as significant XP-EHH values when Andeans were compared to others. One of these SNPs, rs13411586, is differentially expressed in humans; our prediction showed that individuals homozygous for the putatively selected allele (C) have increased Sp100 production.

Interestingly, we also identified that the \textsl{SP100} gene is differentially expressed in skeletal muscles (Supplementary Figure \ref{fig:SciRep_figS3}). Studies have revealed that a member of the HIF pathway, HIF-1, plays an important role in the regulation of oxygen homeostasis, which includes the physiological skeletal and heart muscle adaptations in situations of oxygen reduction due to muscular effort \cite{semenza_regulation_1999,vogt_einfluss_2003,lindholm_skeletal_2016} and ischemic cardiomyopathy, respectively \cite{herrer_gene_2015}. Exposure to high altitude leads to reduced muscle mass and performance (\textit{e.g}. lower work capacity and standing fatigue), except when one is evolutionarily adapted to it \cite{marconi_work_2006,brutsaert_high-altitude_2008,counter_assessment_2017}.

HIF-1 protects cell-survival during low oxygen supply, while p53 promotes genome cell-death under hypoxia. The reason for these apparently antagonistic roles can be in the difference of the oxygen quantity available; in a normal condition, both p53 and HIF-1 levels are low, but in mild hypoxia, the p53 level remains low, whereas the HIF-1 level increases, protecting cells still relatively healthy from destruction. In severe hypoxia, p53 accumulation promotes the repression or degradation of anti-apoptotic proteins like HIF-1, inducing apoptosis of the damaged cells \cite{schmid_hif-1_2004,obacz_cross-talk_2013,zhou_modeling_2015}. Sp100 is known as a modulator of the p53 activity \cite{berscheminski_sp100a_2016} and under tissue hypoxia due to ischemia, it is downregulated, leading to genomic instability \cite{herrer_gene_2015}. The Andean population presents high allele C (rs13411586) frequency (Table \ref{tab:SciRep_table2}), which in homozygosis increase Sp100 production according to our prediction test. Our result suggests an evolutionary solution to keep Sp100 at an adequate level in an environment with a constant low oxygen level. Furthermore, it is possible to speculate that there is an intricate balance in the level of expression of the \textsl{SP100}, \textsl{TP53} and \textsl{HIF-1} genes under hypoxia, considering both short (reversible physiological and metabolic adaptations) and long-term evolutionary adaptation scenarios.

\textsl{DUOX2}, expressed in epithelial cells of various tissues including nasal and lung, participates in the hydrogen peroxide (H\textsubscript{2}O\textsubscript{2}) pathway, which is required in the final steps of thyroid hormones production. It is also involved in Reactive Oxygen Species (ROS), a byproduct of the normal oxygen metabolism even under normal physiologic conditions \cite{devasagayam_free_2004}. However, different stressor conditions can increase the ROS production, \emph{i.e.} high-altitude exposure (hypoxia and UV exposure), and pathological conditions such as cancer \cite{gupta_upsides_2012}. \citeonline{salmeen_nadph_2010} provided evidence that \textsl{DUOX2} plays a role in a p53-dependent checkpoint mechanism for cell cycle entry.

\textit{In vitro} and \textit{in vivo} experiments showed that oxidative stress and generation of ROS caused by \textsl{DUOX2} overexpression, in both hypoxia and hyperoxia, contribute to inflammation, carcinogenesis and cell death \cite{kim_dual_2014,bautista-ortega_supplemental_2014,fletcher_nicotinamide_2014,dias-freitas_molecular_2016,min_ros-dependent_2017,lin_high_2017,macfie_duox2_2014}. For instance, a functional study \cite{kim_dual_2014} showed that under hyperoxia conditions, mutant mice for \textsl{DUOX2} had significant lower acute lung injuries induced by hyperoxia. This finding pointed to the importance of these proteins in the response to changes of oxygen concentration in the environment. Another study \cite{bautista-ortega_supplemental_2014} found that chickens submitted to hypoxia (>3,000 m) had increased activity of DUOX/NOX proteins, indicating the physiological role of these enzymes in the process of adaptation to oxidative stress.

Our results on the expression of the \textsl{DUOX2} putatively selected allele G (high PBS values and significant XP-EHH value > 2; Table \ref{tab:SciRep_table1}) also pointed to higher levels of protein expression in humans, mainly in the lungs and arteries. It is noteworthy that ROS contributes to inflammation in the vessel walls. \cite{kim_oxidative_2014} demonstrated that ROS has an important role in angiogenesis, a process of new blood vessel growth. Angiogenesis is a key event in the physiological response to hypoxia and therefore might have a role in the adaptation to high altitude in long-term residents, especially in individuals with excessive erythropoiesis (like those found in the Chronic Mountain Sickness [CMS] phenotype), to compensate a plausible change in microcirculation \cite{ge_b-type_2011,buroker_akt3_2012}.

SNP rs440191 is located at the 3'UTR region of \textsl{CLC}, and the putatively selected allele G is in complete linkage disequilibrium with the \textsl{CLC} rs395892 G allele in the Mexican population \cite{the_1000_genomes_project_consortium_integrating_2014}. The latter is associated with eosinophil and basophil counts \cite{1000_genomes_project_consortium_global_2015}, while rs440191 has so far been investigated just in approaches assessing allergic susceptibilities \cite{astle_allelic_2016}. Gene expression queries did not show any significant eQTL related to this polymorphism, preventing any prediction of tissue-specific expression.

\textsl{CLC} (galectin-10) is still a poorly studied gene when compared to other members of the functionally polyvalent galectin family. It is recognized as a lysophospholipase expressed in eosinophils and basophils, although some authors identified it just as an enzyme that interacts with lysophospholipases \cite{ackerman_charcot-leyden_2002}. The only functional study regarding this protein showed that hypoxia increases eosinophil accumulation and \textsl{CLC} production in humans, concomitant with a delay in constitutive apoptosis, antagonizing the normal pro-apoptotic effect of agents that normally induce eosinophil apoptosis \cite{porter_hypoxia_2017}.

Regulation by the p53 transcription factor seems to be important in the galectin family genes’ expression. For instance, the galectin-3 gene has a binding site for p53, and p53 increases the transcription of paralogue galectin-7 \cite{polyak_model_1997,raimond_second_1995,cooper_galectinomics_2002}. Altered expression of galectin genes, including \textsl{CLC}, was implicated in cancer emergence and progression, highlighting the role of the galectins in cell proliferation via cell death programs \cite{gopalan_expression_2016}.

Investigations with galectin paralogues have shown that galectin-1 in the first term ovine gestation placenta prevented inflammatory processes that harm the fetus \cite{iglesias_galectin-1_1998}, while galectin-13, which has the highest homology to \textsl{CLC}, is a member of the group of the so-called “pregnancy-related proteins”, due to its special immune functions at the feto-maternal interface \cite{than_functional_2004,su_galectin-13_2018}. These fundamental cell functions, already described for humans and other placental mammals, may indicate the path that connects our \textsl{CLC} findings and the selection pressure in the Andean hostile climate.

In conclusion, our results pointed to a complex adaptation that occurred in Andean natives, which involved the \textsl{CLC}, \textsl{SP100} and \textsl{DUOX2} genes, not previously correlated in contexts of long-time adaptation to high altitudes. We also reinforced the role of the \textsl{TP53} pathway at least for the adaptation to the Andean environmental stresses. Combined with other studies, and incorporating the present one, it is clear that Andeans have adapted to the Altiplano in different ways and using distinct molecular strategies than those of other natives living at high altitude.

\subsection{Methods}

\subsubsection{Populations}

We analyzed 213,987 SNPs determined with Illumina 610quad from 63 Native Americans living at extreme high altitude ($\geq$4,000 m; 63\% of the oxygen available at sea level; \url{http://www.altitude.org/air_pressure.php}) and 259 living at lowland areas ($<$2,500 m), data previously published by \citeonline{reich_reconstructing_2012}. Highlanders included Aymara and Quechua Andeans, while lowlanders were represented by 25 populations from the Mesoamerican and South American lowlands. Details about these populations, sample sizes and allelic frequencies are given in Suplementarry Table \ref{tab:SciRep_tableS2}. Additional information, including ethical authorizations for evolutionary and anthropological studies, can be found in the primary publication58.

\subsubsection{Population Branch Statistic (PBS) analysis}

PBS determinations were performed between pair of populations, using Andean and Amazonian populations as sister groups and Mesoamericans as an outgroup. The analysis was carried out as described by \citeonline{yi_sequencing_2010}, with only the polymorphic SNPs in at least two of the populations being considered. From the genetic distances (FST) between the three population groups examined, PBS measures if there are alleles with extreme frequencies in the Andean group as compared to the other two. Under a scenario of genetic drift only, we expect that Andeans and Amazonians will be more similar genetically than both compared to Mesoamericans. If, however, there has been local adaptation, we should detect genes that have been targeted by selection in Andeans. PBS values were estimated for both individual SNPs and windows of 20 SNPs overlapped in five SNPs. The empiric distribution of PBS values, with a 99.5\textsuperscript{th} threshold, was used to determine signals of positive selection (more details in \citeonline{amorim_genetic_2017}).

\subsubsection{Demographic simulations}

To verify the significance of the observed positive selection signals we simulated different demographic models, according to reported historical population data and inferred effective population sizes. We adapted the models described by \citeonline{valverde_novel_2015}, to account for the divergence between Mesoamericans, Andeans and Amazonians. Assuming that the American continent was peopled beginning at 15,000 yBP, the Andes colonized by 12,000 yBP and the Amazon by 10,000 yBP, and based on Nes estimated by \citeonline{valverde_novel_2015}, we simulated the three demographic models proposed by them: (a) Constant Model: Ne of 7,000 individuals with constant size in all populations throughout history; (b) Bottleneck Model: Ne 8,000 in Mesoamerica, 4,000 in Andes and 2,000 in Amazon; and (c) Bottleneck + Expansion Model: model b with bottlenecks reducing the effective size of all populations by 50\% in the last 10,000 years followed by a sharp expansion in the last 8,000 years. Simulations were performed in the MS program \cite{hudson_generating_2002} with 10,000 replicates for each demographic scenario.

\subsubsection{Linkage disequilibrium analysis}

We also used three linkage disequilibrium-based methods: extended haplotype homozygosity (EHH) \cite{sabeti_detecting_2002}, integrated haplotype score (iHS) \cite{voight_map_2006}, and cross-population extended haplotype homozygosity (XP-EHH) \cite{sabeti_genome-wide_2007}. These approaches adopt the same core principle, that an advantageous allele under a hard sweep rise in frequency ─ carrying its neighbor alleles and therefore promoting homozygosity extension ─ quickly enough that recombination is not able to break down the haplotype. EHH statistics calculate the homozygosity rate from a core region (putative allele under selection) to the neutral scenery, \textit{i.e.} the probability that any two randomly chosen chromosomes will be identical by descent, from the core region to a distance x. iHS evaluates the EHH considering both ancestral and derived alleles, and XP-EHH is used to calculate EHH/iHS between populations, therefore controlling for local variation. These tests are complementary; while iHS is better for detecting incomplete sweeps, XP-EHH has more power to detect sweeps near fixation \cite{vitti_detecting_2013}. Both measurements and significance were calculated through the ‘rehh’ R package \cite{gautier_rehh_2012}.

\subsubsection{Geographical analysis}

To evaluate the variants spatial distribution, weighted inverse distance interpolation (IDW) was used to determine cell values using a weighted linear combination of a set of sample points. Weight is a function of the inverse distance \cite{watson_refinement_1985}. The maps were made with the ArcGis 10.5 software and the cartographic base was georeferenced to the World Geodetic System (WGS84).

\subsubsection{Bootstrap simulations}

To verify whether the allele frequencies of the candidate variants under selection are significantly different among extreme high (>4,000 m) and lowland (<4,000 m) populations, we obtained the 95\% confidence intervals of the average allele frequency of the lowland populations by means of 10,000 computer-assisted bootstrap simulations with replacement, considering a sample as having the same size and genotypic proportions observed in the real one. The average allele frequencies from high and lowland populations were obtained by weighing the observed frequencies according to their sample sizes.

\subsubsection{Analysis of gene expression}

We used the Genotype-Tissue Expression Portal (GTEx; \url{https://www.gtexportal.org/home/}) to evaluate possible associations between each of the candidate alleles with highest differentiation and gene expression across human tissues looking for evidence of quantitative trait loci (eQTLs). The m-value is the posterior probability that an eQTL effect exists in each tissue tested in the cross-tissue meta-analysis. The m-value ranges between 0 and 1 (m-values > 0.9 mean that the tissue is predicted to have an eQTL effect).

\clearpage
\subsection{Supplementary Material}

% Table S1
\input{tabelas/SciRep_tableS1}

% Table S2
\input{tabelas/SciRep_tableS2}

% Figure S1
\begin{figure}[!htbp]
    \vspace{2cm}
    \centering
    \includegraphics[width=1.0\linewidth]{SciRep_figS1}
    \caption{iHS value distribution patterns for all three groups (South American highland, South America Lowland and Mesoamerica Lowland).}
    \label{fig:SciRep_figS1}
\end{figure}

% Figure S2
\begin{figure}[!htbp]
    \centering
    \includegraphics[width=1.0\linewidth]{SciRep_figS2}
    \caption[Bootstrap simulations]{Bootstrap simulations. (A) Table showing average values (mean and median) of the allele frequencies for each SNP in lowland populations, as well as, their 95\% confidence intervals obtained by simulation, and the average allele frequency of the candidate variant in highland populations. (B─F). Distribution of allele frequencies obtained by 10,000 simulations for lowlanders considering all markers in putative selection. The corresponding average allele frequencies observed for highland populations are represented by black vertical lines.}
    \label{fig:SciRep_figS2}
\end{figure}

% Figure S3
\begin{figure}[hp]
    \centering
    \includegraphics[width=1.0\linewidth]{SciRep_figS3}
    \caption[Differential expression of the \textsl{DUOX2} putatively selected allele.]{Differential expression of the \textsl{DUOX2} putatively selected allele (rs269866 G) for a differential effect in gene expression in the thyroid tissue \url{(https://www.gtexportal.org/home/}, accessed 26/03/2018).}
    \label{fig:SciRep_figS3}
\end{figure}

% Figure S4
\begin{figure}[hp]
    \centering
    \includegraphics[width=1.0\linewidth]{SciRep_figS4}
    \caption[Posterior probabilities of \textsl{DUOX2} putatively selected allele.]{Posterior probabilities of \textsl{DUOX2} putatively selected allele (rs269866G) for a differential effect in gene expression in multiple tissues (\url{https://www.gtexportal.org/home/}, accessed 26/03/2018).}
    \label{fig:SciRep_figS4}
\end{figure}

% Figure S5
\begin{figure}[hp]
    \centering
    \includegraphics[width=1.0\linewidth]{SciRep_figS5}
    \caption[Differential expression of the \textsl{SP100} putatively selected allele.]{Differential expression of the \textsl{SP100} putatively selected allele (rs13411586C) in the skeletal muscle (\url{https://www.gtexportal.org/home/}, accessed 26/03/2018).}
    \label{fig:SciRep_figS5}
\end{figure}
    
% Figure S6
\begin{figure}[hp]
    \centering
    \includegraphics[width=1.0\linewidth]{SciRep_figS6}
    \caption[Posterior probabilities of \textsl{SP100} putatively selected allele.]{Posterior probabilities of \textsl{SP100} putatively selected allele(rs13411586C) for a differential effect in gene expression in multiple tissues (\url{https://www.gtexportal.org/home/}, accessed 26/03/2018).}
    \label{fig:SciRep_figS6}
\end{figure}

\selectlanguage{brazil}
